.travis.yml

```diff
@@ -69,9 +70,11 @@ script:
   - if [ "$KNAPSACK_PRO_CI_NODE_INDEX" == "1" ]; then bundle exec bundle-audit check --update --ignore CVE-2015-9284; fi
   - if [ "$KNAPSACK_PRO_CI_NODE_INDEX" == "1" ]; then yarn build-storybook; fi
   - if [ "$KNAPSACK_PRO_CI_NODE_INDEX" == "2" ]; then bin/test-console-check; fi
-  - export FOREM_OWNER_SECRET="secret"
+  - export FOREM_OWNER_SECRET="secret" # test secret so e2e tests can run properly.
```

Comment:
same, I think this should be moved up to the `global` section with other environment variables